src/adagrams.js

```diff
@@ -1,15 +1,141 @@
+const letterPoolObj = {
+  A: 9, 
+  B: 2, 
+  C: 2, 
+  D: 4, 
+  E: 12, 
+  F: 2, 
+  G: 3, 
+  H: 2, 
+  I: 9, 
+  J: 1, 
+  K: 1, 
+  L: 4, 
+  M: 2, 
+  N: 6, 
+  O: 8, 
+  P: 2, 
+  Q: 1, 
+  R: 6, 
+  S: 4, 
+  T: 6, 
+  U: 4, 
+  V: 2, 
+  W: 2, 
+  X: 1, 
+  Y: 2, 
+  Z: 1
+}
+
 export const drawLetters = () => {
   // Implement this method for wave 1
+  let letterList = [];
+  let letterPoolCopy = letterPoolObj;
```

Comment:
This doesn't make a copy of letterPoolObj. This line of code means that letterPoolCopy will refer to the same object that letterPoolObj does. So if you change a value in letterPoolCopy, you'll also change the value in letterPoolObj.

You can use the spread operator to make a copy so when you change letterPoolCopy, you don't mutate your orig object:
```
 const letterPoolCopy = {
    ...letterPoolObj
 };